broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/PartitionTransitionImpl.java

```diff
@@ -68,6 +68,10 @@ public ActorFuture<Void> transitionTo(final long term, final Role role) {
     // ongoing activity at this point in time
     steps.forEach(step -> step.onNewRaftRole(context, role));
 
+    // setup transition step monitor
+    context.setIsPartitionInTransition(true);
+    context.setTimeOfLastTransitionStep(System.currentTimeMillis());
```

Comment:
:x: Let's use `ActorClock.currentTimeMillis()`. We should always use the actor clock and not the system clock because it makes testing easier.